### ISSUE TITLE
Draw dependcies between tasks

### ISSUE THREAD

**Issue by Eccenux** [mbielanczuk/jQuery.Gantt#2]
This is something jsGantt has - the dependencies are drawn based on a list of IDs of tasks. See example at  jsgant.com - tasks "Constructor Proc" and "Task Variables".

BTW ID would be nice to have as you can generate basic Gantt from Tasks added to something like Flyspray ;-) or Bugzilla.

**Comment by taitems** [comment]
Clickable: <a href="http://jsgantt.com/" target="_blank">http://jsgantt.com/</a>

I can't image an easy way of doing this cross browser without SVG degrading to VML. Otherwise it just becomes tag soup with table cells etc.

**Comment by Eccenux** [comment]
I agree that you cannot do this really cleanly, but you don't have to use table cells. If you know position and dimensions of point A and point B then drawing a line between them is not very hard. Drawing arrows is a bit hacky in jsGantt, but in real life scenarios task B should always depend on task A so an arrow is not really needed.

I've prepared a quick HTML example here:
http://enux.pl/smp/draw_dep/
